app/javascript/crayons/guidelines/__stories__/Components.stories.mdx

```diff
@@ -19,14 +19,14 @@ Preact.
 The entire design system uses the [BEM](http://getbem.com/naming) methodology
 for naming CSS classes. Even Preact components under the hood use it.
 
-### `crayons-*`
+### `crayons-*` or `c-*`
```

Comment:
Should we maybe add a note to this section that references to `crayons-` will eventually be superseded by `c-`? Just to be clear which is the preferred prefix at the moment.

Reply:
good suggestion. adding.